galvan-support/src/main/java/org/terracotta/testing/rules/BasicExternalCluster.java

```diff
@@ -63,6 +64,7 @@ public BasicExternalCluster(File clusterDirectory, int stripeSize, List<File> se
     this.stripeSize = stripeSize;
     this.namespaceFragment = namespaceFragment;
     this.serviceFragment = serviceFragment;
+    this.entityFragment = serviceFragment;
```

Comment:
No!  these PRs where discussion points between Jeff and I.  I need to do a few updates before merge.  Thanks!